crates/bevy_asset/src/ser.rs

```diff
@@ -75,31 +75,14 @@ impl<'de, T: Asset> Deserialize<'de> for Handle<T> {
 }
 
 impl AssetServer {
-    pub fn serialize_with_asset_refs<S, T>(
-        &self,
-        serializer: S,
-        value: &T,
-    ) -> Result<S::Ok, S::Error>
+    /// Enables asset references to be serialized or deserialized
+    pub fn with_asset_refs_serialization<F, T>(&self, f: F) -> T
```

Comment:
Could you give `f` a more descriptive name?

Reply:
`f`, `func` are commonly used in this situations, anything in mind?

Reply:
Something like `serde_func` (that expresses the functions does some sort of (de)serialization would be nice.